lib/ui/base/text_styles.dart

```diff
@@ -177,8 +177,16 @@ class TextStyles {
 
   static const TextStyle searchBarText =
       TextStyle(fontFamily: 'OpenSans', fontSize: 14, color: Color(0xFF192123));
-  static const TextStyle searchBarHint =
-      TextStyle(fontFamily: 'OpenSans', fontSize: 14, color: ColorsPlante.grey);
+  static const TextStyle searchBarHint = TextStyle(
+      fontFamily: 'OpenSans',
+      fontSize: 14,
+      color: Colors.grey,
+      fontWeight: FontWeight.bold);
```

Comment:
Hmm, are you sure the `searchBarHint` style should be bold?
I've checked in Figma - it's still regular, not bold

Reply:
no, my bad, i made a copy of searchbarhint.